.github/workflows/versions.yml

```diff
@@ -51,6 +51,46 @@ jobs:
           __tests__/verify-node.sh "${BASH_REMATCH[1]}"
         shell: bash
 
+  nightly-syntax:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: [17-nightly, 18-nightly, 19-nightly]
```

Comment:
I notice we test the version range syntax here and exact versions for RC.  But, we support both for both cases, right?

Reply:
For the rc versions it's required to specify the full version. Semver does not support range for rc versions that is why we use the same behaviour as for setup-go https://github.com/actions/setup-go. Besides, nightly syntax is not included to SemVer that is why we use similar approach as for setup-python pypy: https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#specifying-a-pypy-version.

Reply:
Ok.  Do we want to test an exact version for nightly at least?